02 - Community Expansions/02.01 Plugins by Category/🗂️ 02.01 Plugins by Category.md

```diff
@@ -9,50 +9,147 @@ tags:
 
 #placeholder/description 
 
+## Categories
+
+### Language support
+
+- [[Chinese Language Plugins]]
+- [[Japanese Language Plugins]]
+- [[Plugins for RTL Languages]]
+- [[Dictionary and Spellchecking Plugins]]
+
+### UX plugins
+
+- [[Status bar plugins]]
+- [[Theme settings plugins]]
+- [[Plugins Designed for Mobile]]
+- [[Style and Appearance Plugins]]
+- [[Side bar plugins]]
+- [[Pane Management plugins]]
+- [[Plugins that add or manage hotkeys]]
+- [[Toolbar plugins]]
+
+### Navigation & search plugins
+
+- [[Plugins to navigate notes]]
+- [[Search and query plugins]]
+- [[Plugins to Enhance Workspaces]]
+- [[Folder management plugins]]
+- [[Tag management plugins]]
+
+### File Management plugins
+
+- [[Plugins to manage files and attachments]]
+- [[Plugins for Video and Audio files]]
+- [[Plugins to export and import markdown files]]
+
+### Note contents
+
+- [[Plugins to manage or annotate PDF files]]
+- [[Plugins to handle images]]
+- [[Mindmapping plugins]]
+- [[Plugins to create charts]]
+- [[Plugins for Diagrams]]
+- [[Plugins with custom codeblock syntax]]
+- [[Plugins with custom views]]
+
+### Editor plugins
+
+- [[Plugins for Editing Notes]]
+- [[Plugins to edit other file types]]
+- [[Refactoring and auto-formatting plugins]]
+- [[Markdown formatting plugins]]
+- [[Vim-related Plugins]]
+- [[Template plugins]]
+
+### Integration
+
+- [[Third-Party Integration Plugins]]
+- [[Automation plugins]]
+- [[Backup plugins]]
+- [[Plugins to export and import markdown files]]
+- [[Date and calendar plugins]]
+- [[Plugins for Security and Privacy]]
+- [[Task management plugins]]
+- [[Plugins to manage internal and external links]]
+
+### Workflows
+
+- [[Plugins for Writers]]
+- [[Plugins for daily pages]]
+- [[Spaced Repetition Plugins]]
+
+### Miscellaneous
+
+- [[Plugins for Chess]]
+- [[Plugins for TTRPG]]
+- [[Paid and subscription-based plugins]]
+- [[Finance plugins]]
+- [[Mathjax and LaTeX Plugins]]
+- [[Uncategorized plugins]]
+- [[Unlisted plugins]]
+
+
+
 ## MOC
```

Comment:
is the MOC needed, when they are all basically already listed above?

Reply:
Might be removed once we disable zoott